src/interface/common/makeAnalyzerUrl.js

```diff
@@ -35,6 +35,9 @@ export default function makeReportUrl(report = undefined, fightId = undefined, p
             parts.push(duplicatePlayerNames.length > 1 ? `${playerId}-${prettyEncodeURI(playerName)}` : prettyEncodeURI(playerName));
             if (tab) {
               parts.push(tab);
+              if(build){
+                parts.push(build);
```

Comment:
just a thought, maybe it should be build/tab rather than tab/build ? I think 'build' is of bigger scope

Reply:
just pushed a change for that but I ran into slight issues when there's no build / no selected one, gonna try to fix those tomorrow probably